### PR TITLE
BOTMETA: add maintainer, support and labels for hwc_utils.py and unit…

### PR DESCRIPTION
**Pull request by zengchen1024**
##### SUMMARY
add maintainer, support and labels for hwc_utils.py and unit test

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
botmeta

### CODE REVIEW

File: .github/BOTMETA.yml [hunk 1]
```diff
@@ -689,7 +689,12 @@ files:
     maintainers: $team_google
     supershipit: $team_google
   $module_utils/gitlab.py: *gitlab
-  $module_utils/hwc_utils.py: *huawei
+  $module_utils/hwc_utils.py:
```
Comment (acozine) [inline .github/BOTMETA.yml]:
```suggestion
  $module_utils/hwc_utils.py: &huawei
```
If you add an anchor here . . .

File: .github/BOTMETA.yml [hunk 2]
```diff
@@ -1418,7 +1423,12 @@ files:
     <<: *docker
     support: community
   test/units/module_utils/facts/network/test_generic_bsd.py: *bsd
-  test/units/module_utils/hwc: *huawei
+  test/units/module_utils/hwc:
```
Comment (acozine) [inline .github/BOTMETA.yml]:
```suggestion
  test/units/module_utils/hwc: *huawei
```
. . . you can refer to it with an alias here and get rid of the duplicated lines.

File: .github/BOTMETA.yml [hunk 3]
```diff
@@ -1418,7 +1423,12 @@ files:
...
+    maintainers: $team_huawei
```
Comment (acozine) [inline .github/BOTMETA.yml]:
```suggestion
```

File: .github/BOTMETA.yml [hunk 4]
```diff
@@ -1418,7 +1423,12 @@ files:
...
+    support: community
```
Comment (acozine) [inline .github/BOTMETA.yml]:
```suggestion
```

File: .github/BOTMETA.yml [hunk 5]
```diff
@@ -1418,7 +1423,12 @@ files:
...
+    labels:
```
Comment (acozine) [inline .github/BOTMETA.yml]:
```suggestion
```

File: .github/BOTMETA.yml [hunk 6]
```diff
@@ -1418,7 +1423,12 @@ files:
...
+      - networking
```
Comment (acozine) [inline .github/BOTMETA.yml]:
```suggestion
```

File: .github/BOTMETA.yml [hunk 7]
```diff
@@ -1418,7 +1423,12 @@ files:
...
+      - huawei
```
Comment (acozine) [inline .github/BOTMETA.yml]:
```suggestion
```

**Comment by zengchen1024** [comment]
@acozine If I understand correctly. you means this pr is no need, right? Because it seems that your comments want to me to revert the modifications.  But @samdoran asked me to add this pr. I am a little confused. sorry!

**Comment by acozine** [comment]
@zengchen1024 thanks for responding - there is a need for this PR. However, you don't need to repeat lines 693-697 on lines 1427-1431. If you create the anchor in the first section (line 692), you can refer to it in the second section (line 1426) without duplicating all the information. 

See https://docs.ansible.com/ansible/devel/user_guide/playbooks_advanced_syntax.html?highlight=anchors#yaml-anchors-and-aliases-sharing-variable-values for some documentation on using YAML anchors and aliases.

**Comment by zengchen1024** [comment]
@acozine Thank very much for your detailed explanations. I have updated it as your suggestions.

**Comment by acozine** [comment]
Thanks @zengchen1024.

**Pull request merged**